README.md

```
@@ -74,41 +74,33 @@ claude plugin marketplace add norrietaylor/distillery
-      "env": {
-        "JINA_API_KEY": "${JINA_API_KEY}"
-      }
-    }
-  }
-}
-```
+`uvx` inherits this from your shell environment. Without a key, Distillery falls back to a stub embedding provider (search quality degraded).
 
 Restart Claude Code and run the onboarding wizard:
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)